### ISSUE TITLE
Could not load SsbhLib

### ISSUE THREAD

**Issue by EthanWeegee** [Ploaj/StudioSB#59]
StudioSB attempts to load SsbhLib, but the library is named `SSBHLib.dll`. Simply renaming the library to `SsbhLib.dll` fixes this.

**Comment by ScanMountGoat** [comment]
I don't get this error when downloading the latest release. Are you trying to build from source?

**Comment by EthanWeegee** [comment]
No, I'm running via Mono. Filesystem is case-sensitive, hence the error.

**Comment by ScanMountGoat** [comment]
I doubt the program will run correctly under mono even if winforms is partially supported.

**Comment by EthanWeegee** [comment]
Seems to run fine, actually.
